src/Components/QuestionList/QuestionListS.scss

```diff
@@ -3,60 +3,12 @@
 .QuestionList {
     display: flex;
     flex-direction: column;
-    width: 70vw;
+    width: 81vw;
     height: 80vh;
```

Comment:
I'm not sure that you want to do this except for at the app level. This can get complicated if added too many places

Reply:
changed to %